AFQ/tasks/data.py

```diff
@@ -311,53 +398,206 @@ def dki_awf(subses_dict, dwi_affine, dki_params_file, dki_tf,
 @as_file('_model-DKI_MK.nii.gz')
 @as_dt_deriv('DKI')
 def dki_mk(subses_dict, dwi_affine, dki_params_file, dki_tf):
+    """
+    full path to a nifti file containing
+    the DKI mean kurtosis file
+    """
     return dki_tf.mk()
 
 
 @pimms.calc("dki_ga_file")
 @as_file(suffix='_model-DKI_GA.nii.gz')
 @as_dt_deriv(tf_name='DKI')
 def dki_ga(subses_dict, dwi_affine, dki_params_file, dki_tf):
+    """
+    full path to a nifti file containing
+    the DKI geodesic anisotropy
+    """
     return dki_tf.ga
 
 
 @pimms.calc("dki_rd_file")
 @as_file(suffix='_model-DKI_RD.nii.gz')
 @as_dt_deriv(tf_name='DKI')
 def dki_rd(subses_dict, dwi_affine, dki_params_file, dki_tf):
+    """
+    full path to a nifti file containing
+    the DKI radial diffusivity
+    """
     return dki_tf.rd
 
 
 @pimms.calc("dki_ad_file")
 @as_file(suffix='_model-DKI_AD.nii.gz')
 @as_dt_deriv(tf_name='DKI')
 def dki_ad(subses_dict, dwi_affine, dki_params_file, dki_tf):
+    """
+    full path to a nifti file containing
+    the DKI axial diffusivity
+    """
     return dki_tf.ad
 
 
 @pimms.calc("dki_rk_file")
 @as_file(suffix='_model-DKI_RK.nii.gz')
 @as_dt_deriv(tf_name='DKI')
 def dki_rk(subses_dict, dwi_affine, dki_params_file, dki_tf):
+    """
+    full path to a nifti file containing
+    the DKI radial kurtosis
+    """
     return dki_tf.rk
 
 
 @pimms.calc("dki_ak_file")
 @as_file(suffix='_model-DKI_AK.nii.gz')
 @as_dt_deriv(tf_name='DKI')
 def dki_ak(subses_dict, dwi_affine, dki_params_file, dki_tf):
+    """
+    full path to a nifti file containing
+    the DKI axial kurtosis file
+    """
     return dki_tf.ak
 
 
-def get_data_plan(brain_mask_definition):
+@pimms.calc("brain_mask_file")
+@as_file('_brain_mask.nii.gz')
+def brain_mask(subses_dict, dwi_affine, b0_file,
+               bids_info, brain_mask_definition=None):
+    """
+    full path to a nifti file containing
+    the brain mask
+
+    Parameters
+    ----------
+    brain_mask_definition : instance from `AFQ.definitions.mask`, optional
+        [REGISTRATION] This will be used to create
```

Comment:
```suggestion
        This will be used to create
```
Is this still needed?